x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/_legend.scss

```diff
@@ -28,11 +28,20 @@
   display: 'inline-block';
 }
 
-.mlJobMapLegend__inferenceModel {
+.mlJobMapLegend__trainedModel {
   height: $euiSizeM;
   width: $euiSizeM;
   background-color: '#FFFFFF';
   border: 1px solid $euiColorMediumShade;
   border-radius: 50%;
   display: 'inline-block';
 }
+
+.mlJobMapLegend__sourceNode {
+  height: $euiSizeM;
+  width: $euiSizeM;
+  background-color: $euiColorLightShade;
+  border: 1px solid $euiColorMediumShade;
+  border-radius: 50%;
```

Comment:
Again, lacking context, this might be replaceable with an `$euiBorderRadius` unless you are going for some rounder effect.

Reply:
Updated in 0e95f8993b8184dfdf3f02eaad3b476eda9ac79a